src.mk

```diff
@@ -294,6 +294,22 @@ LIB_SOURCES =                                                   \
   utilities/ttl/db_ttl_impl.cc                                  \
   utilities/write_batch_with_index/write_batch_with_index.cc    \
   utilities/write_batch_with_index/write_batch_with_index_internal.cc    \
+  cloud/aws/aws_env.cc                                          \
```

Comment:
This change is the problem. aws_env.cc and others are listed twice in LIB_SOURCES now.